README.md

```diff
@@ -9,7 +9,8 @@
 
 `<model-viewer>` is a web component that makes rendering interactive 3D
 models - optionally in AR - easy to do, without writing any code, on as many
-browsers and devices as possible.
+browsers and devices as possible. This component will strive to have great
```

Comment:
"`<model-viewer>` strives to give you great defaults for quality and performance."

Unrelated, but the phrase "without writing any code" is kind of weird. I would be willing to speculate that a lot of the folks who are being targeted with that phrase would consider HTML to be code.